content/en/blog/2025/ambient-multicluster/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Introducing multicluster support for ambient mode (alpha)
+description: Introducing multicluster support for ambient mode (alpha).
```

Comment:
These should be different from each other; description is probably optional

Reply:
I recall a dumb lint rule that will complain if one of these doesn't end with a period